talk/owt/sdk/p2p/p2pclient.cc

```diff
@@ -293,24 +293,40 @@ void P2PClient::OnSignalingMessage(const std::string& message,
         rtc::GetIntFromJsonObject(stop_info, "code", &code);
         rtc::GetStringFromJsonObject(stop_info, "message", &error);
 
+        if (code == kWebrtcIceGatheringPolicyUnsupported) {
+          auto pcc = that->GetPeerConnectionChannel(remote_id);
```

Comment:
Please check validity of  pc channel before you use it.

Reply:
`GetPeerConnectionChannel` creates a new PeerConnectionChannel if there is no available channel associated with `remote_id`, so `pcc` should always be valid here.